TwilioVerifySDK/TwilioVerify/Sources/TwilioVerifyConfig.swift

```diff
@@ -17,7 +17,7 @@
 //  limitations under the License.
 //
 
-let version = "2.2.2"
+let version = "2.3.0"
```

Comment:
This will be updated by the pipeline, please revert the change. We should use semantic versioning when merging this PR, this is a feature.